File: yeji/week17/1002.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace baek2
{
    class Program
    {
        public static void Main()
        {
            int n = int.Parse(Console.ReadLine());
            while (n > 0)
            {
                n--;
                string[] s = Console.ReadLine().Split();

                int x1 = int.Parse(s[0].ToString());
                int y1 = int.Parse(s[1].ToString());
                int r1 = int.Parse(s[2].ToString());
                int x2 = int.Parse(s[3].ToString());
                int y2 = int.Parse(s[4].ToString());
                int r2 = int.Parse(s[5].ToString());

                int sub = Math.Abs(r1 - r2);
                double d = Math.Pow(MathF.Pow(x2 - x1, 2) + MathF.Pow(y2 - y1, 2),0.5);

                if (d == 0 && r2 == r1) Console.WriteLine(-1);
                else if (d < r1 + r2 && d > sub) Console.WriteLine(2);
                else if (r1 + r2 == d || sub == d) Console.WriteLine(1);
                else Console.WriteLine(0);
            }
        }
    }
}

File: jekyoung/week17/1002.py
import sys

T = int(sys.stdin.readline())

for _ in range (T):
    x1, y1, r1, x2, y2, r2 = map(int, sys.stdin.readline().split())
    distance = float(((x1-x2)**2 + (y1-y2)**2)**0.5)
    
    # 원의 중심이 동일할 때,
    if distance == 0:
        # 동일한 원
        if r1 == r2: print(-1)
        # 만나지 않는 두 원
        else: print(0)
        continue

    # 원의 중심이 다를 때,
    sum = r1 + r2
    sub = abs(r1-r2)
    if sum == distance or sub == distance:
        # 접하는 경우
        print(1)
    elif sub < distance < sum:
        # 두 점에서 만나는 경우 
        print(2)
    else:
        # 만나지 않는 경우
        print(0)
